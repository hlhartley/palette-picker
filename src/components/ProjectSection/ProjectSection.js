import React from 'react';
import ProjectControls from '../../containers/ProjectControls/ProjectControls';
import Search from '../../containers/Search/Search';
import ProjectList from '../../containers/ProjectList/ProjectList';

export const ProjectSection = () => {
    return (
        <div className="ProjectSection">
            <ProjectControls/>
            <Search/>
            <ProjectList/>
        </div>
    )
}

export default ProjectSection;