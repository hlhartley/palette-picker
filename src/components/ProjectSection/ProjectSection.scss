.ProjectSection {
    border: 2px solid blue;
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    width: 49%;
}