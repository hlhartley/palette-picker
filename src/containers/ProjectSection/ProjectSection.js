import React from 'react';
import ProjectControls from '../../containers/ProjectControls/ProjectControls';
import Search from '../../containers/Search/Search';
import ProjectList from '../../containers/ProjectList/ProjectList';

export const ProjectSection = (props) => {
    return (
        <div className="ProjectSection">
            <div className="subheading-container">
                <h2 className="project-header">Project Name</h2>
            </div>
            <div className="content-container">
                <ProjectControls/>
                <Search/>
                <ProjectList/>
            </div>
        </div>
    )
}

export default ProjectSection;