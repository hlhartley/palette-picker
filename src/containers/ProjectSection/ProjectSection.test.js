import React from 'react';
import { shallow } from 'enzyme';
import { ProjectSection } from './ProjectSection';

describe('ProjectSection', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <ProjectSection/>
      )
    });
  it('should properly render the component elements', () => {
    expect(wrapper).toMatchSnapshot();
  });
});