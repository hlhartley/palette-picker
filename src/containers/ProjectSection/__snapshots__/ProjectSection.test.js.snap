// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectSection ProjectSection component should properly render the component elements 1`] = `
<div
  className="ProjectSection"
>
  <div
    className="subheading-container"
  >
    <h2
      className="project-header"
    >
      NO PROJECT LOADED
    </h2>
  </div>
  <div
    className="content-container"
  >
    <Connect(ProjectControls) />
    <hr />
    <Search />
    <hr />
    <Connect(ProjectList) />
  </div>
</div>
`;
